eth2near/eth2-contract-init/src/config.rs

```diff
@@ -0,0 +1,112 @@
+use contract_wrapper::eth_network_enum::EthNetwork;
+use contract_wrapper::near_network_enum::NearNetwork;
+use contract_wrapper::near_rpc_client::NearRPCClient;
+use reqwest::Url;
+use serde::Deserialize;
+use std::io::Read;
+use std::path::PathBuf;
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct Config {
+    // endpoint to a full node of Eth2 Beacon chain with Light Client API
+    pub beacon_endpoint: String,
+
+    // endpoint for the Ethereum full node, which supports Eth1 RPC API
+    pub eth1_endpoint: String,
+
+    // endpoint for a full node on the NEAR chain
+    pub near_endpoint: String,
+
+    // Account id from which relay make requests
+    pub signer_account_id: String,
+
+    // Path to the file with a secret key for signer account
+    pub path_to_signer_secret_key: String,
+
+    // Account id for eth client contract on NEAR
+    pub contract_account_id: String,
+
+    // The Ethereum network name (mainnet, kiln, ropsten, goerli)
+    pub ethereum_network: EthNetwork,
+
+    // NEAR network name (mainnet, testnet)
+    pub near_network_id: NearNetwork,
+
+    // Path to dir for output submitted light client updates and execution blocks
+    pub output_dir: Option<String>,
+
+    // Timeout for ETH RPC requests in seconds
+    pub eth_requests_timeout_seconds: Option<u64>,
+
+    pub validate_updates: Option<bool>,
+
+    pub verify_bls_signature: Option<bool>,
+
+    pub hashes_gc_threshold: Option<u64>,
+
+    pub max_submitted_blocks_by_account: Option<u32>,
+
+    pub trusted_signature: Option<String>,
```

Comment:
Should be `trusted_signer_account_id`

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/825/commits/c924a32cd6d7bf0ee1a7d54532ebc89e25ff6f48